### PR TITLE
i18n albanian language

### PR DESCRIPTION
**Pull request by aldialimucaj**
-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [ ] Tests are added where necessary
-   [X] Documentation is added/updated where necessary
-   [X] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by DanielFran** [comment]
Restarted travis tests...

File: generators/generator-constants.js [hunk 1]
```diff
@@ -100,6 +100,11 @@ const SQL_DB_OPTIONS = [
 ];
 
 const LANGUAGES = [
+    {
+        name: 'Albanian',
+        dispName: 'Shqip',
+        value: 'al'
+    },
```
[the remark this answers is not in the document]
Comment (DanielFran) [reply generators/generator-constants.js]: Checked in https://momentjs.com page and the value for Albanian seems 'sq'.

**Comment by DanielFran** [comment]
@aldialimucaj Can you please update the PR with the changes requested by @ruddell ?
Add  `momentLocaleId: 'sq'`
Thanks

**Comment by DanielFran** [comment]
Thanks @aldialimucaj! Congratulation for your 1st contribution to this project.

**Comment by DanielFran** [comment]
@aldialimucaj  : can you do a PR against https://github.com/jhipster/jhipster.github.io please?

https://github.com/jhipster/jhipster.github.io/blob/master/pages/installing_new_languages.md

File: generators/generator-constants.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (aldialimucaj) [reply generators/generator-constants.js]: Thanks guys. sorry for the little delay.

**Pull request merged**